src/Views/Homepage.vala

```diff
@@ -99,9 +99,13 @@ public class AppCenter.Homepage : AbstractView {
             column_spacing = 24,
             orientation = Gtk.Orientation.VERTICAL
         };
+#if PACKAGEKIT_BACKEND
```

Comment:
I don't understand these changes to `Homepage`. The banners/carousels shouldn't have any dependency on PackageKit as they display Flatpak packages anyway. So a PackageKit related build flag shouldn't be affecting things that don't relate to PackageKit. What's the issue you're seeing while these lines are still enabled?

Reply:
Sorry, I've just seen your description in the PR description. I think this probably needs some debugging rather than just disabling this code, because this is unrelated to PackageKit.

Reply:
Actually I also think this is weird and maybe I need someone to reproduce the issue I mentioned above (as shown in the last screenshot)...🤯 @davidak Can you try?

Reply:
I guess I will just revert this for now...🤦

Reply:
Is it possible not to remove `~/.cache/io.elementary.appcenter/flatpak-metadata/*` on every launch?

https://github.com/elementary/appcenter/blob/aa29793377be481c02a7b88cf3aa1e398a4d4f16/src/Core/FlatpakBackend.vala#L665